src/Microsoft.ML/Models/OneVersusAll.cs

```diff
@@ -13,8 +13,20 @@ namespace Microsoft.ML.Models
     public sealed partial class OneVersusAll
     {
         /// <summary>
-        /// Create OneVersusAll multiclass trainer.
+        /// One versus all learner (also known as One vs the rest) is a multiclass learner with
+        /// strategy to fit one classifier per class. For each class it trains the provided 
+        /// binary classifier against all the other classes. 
+        /// See <a href="https://en.wikipedia.org/wiki/Multiclass_classification#One-vs.-rest">wikipedia</a> page.
         /// </summary>
+        ///<example>
+        /// In order to use it all you need to do is add it to pipeline as regular learner:
+        /// 
+        /// pipeline.Add(OneVersusAll.With(new StochasticDualCoordinateAscentBinaryClassifier()));
+        /// </example>
+        /// <remarks>
+        /// Underlying trainer suppose to be binary classfier. To check your options type BinaryClassifier
+        /// and look on all available learners suggested by IntelliSense.
```

Comment:
Minor recommendations to wording/spelling:
"The base trainer must be a binary classifier. To check the available binary classifiers, type "BinaryClassifiers", and look at the available binary learners as suggested by IntelliSense."